.github/workflows/sync-testnets.yml

```diff
@@ -107,6 +107,7 @@ jobs:
       - name: Display Docker logs        
         working-directory: sedge
         run: docker compose logs
+        if: always()
```

Comment:
Have you seen [this](https://docs.github.com/en/actions/learn-github-actions/expressions#always)? Perhaps not your case but take a look anyway.

Reply:
Good point, wasn't aware of this